### DISCUSSION TITLE
Is it an issue to remove "engines" in the template?

### DISCUSSION THREAD

**Discussion by dcangulo** [callstack/react-native-builder-bob#512] [Q&A] [answered]
The React Native library already has `engines` in the template that force you to use a specific range of node versions.

* [0.71: >=14](https://github.com/facebook/react-native/blob/0.71-stable/package.json#L9)
* [0.72: >=16](https://github.com/facebook/react-native/blob/0.72-stable/packages/react-native/package.json#L9)
* [0.73: >=18](https://github.com/facebook/react-native/blob/0.73-stable/packages/react-native/package.json#L24)

Currently, the `create-react-native-library` uses `>= 18.0.0` in the template. 

This means that the library created by `create-react-native-library`  will force users who are still on 0.71 and 0.72 to use Node.js 18 or they will encounter the following error:
```
The engine "node" is incompatible with this module. Expected version ">= 18.0.0". Got "14.21.3"
```

But 0.71 and 0.72 are still officially supported based on the [support policy](https://github.com/reactwg/react-native-releases/blob/main/README.md#releases-support-policy).

Is it an issue to remove `engines` in `create-react-native-library` template and just let the React Native core handle it?

**Comment by satya164** [comment, marked as answer]
Sounds reasonable to remove it from the template.

Note that the template doesn't follow React Native's support policy. The goal is to support as many versions as it can for legacy arch, and only latest version with new arch. Though we only test with the latest version.
